src/modules/user/actions.js

```diff
@@ -0,0 +1,5 @@
+import { createAction } from 'redux-actions';
+
```

Comment:
I'd prefer you to use vanilla redux so you could understand how it works on practice